includes/class-sensei-list-table.php

```diff
@@ -200,26 +200,25 @@ function get_column_info() {
 	 */
 	function single_row( $item ) {
 		static $row_class = '';
-		$row_class = ( $row_class == '' ? ' class="alternate"' : '' );
+		$row_class = ( $row_class == '' ? 'alternate' : '' );
 
-		echo '<tr' . $row_class . '>';
+		echo '<tr class="' . esc_attr( $row_class ) . '">';
 
 		$column_data = $this->get_row_data( $item );
 
 		list( $columns, $hidden ) = $this->get_column_info();
 
 		foreach ( $columns as $column_name => $column_display_name ) {
-			$class = "class='$column_name column-$column_name'";
-
 			$style = '';
-			if ( in_array( $column_name, $hidden ) )
-				$style = ' style="display:none;"';
 
-			$attributes = "$class$style";
+			if ( in_array( $column_name, $hidden ) )
```

Comment:
We should have braces for this `if` statement.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/a7c63240863f3021097e38d830b8d574867ea2ae.